### DISCUSSION TITLE
External macro implementation type 'MacroPlugin.Codable' could not be found for macro 'Codable()'

### DISCUSSION THREAD

**Discussion by JarnoRFB** [SwiftyLab/MetaCodable#118] [Q&A]
I added MetaCodable to an iOS only project via SwiftPM. I set the version to 1.3 to avoid a swift-syntax dependency conflict with [PowerAssert](https://github.com/kishikawakatsumi/swift-power-assert). 
The compiler fails for the simple example

```
@Codable
enum MyEnum {
    case one
    case two
}
```

with the error message `External macro implementation type 'MacroPlugin.Codable' could not be found for macro 'Codable()'; '.../Products/Debug/MacroPlugin' produced malformed response`.

Is this a known issue?

**Comment by soumyamahunt** [comment]
Can you provide a sample project?
